### PR TITLE
Integration with "Open Food Facts" (partially resolves #422)

### LINKED ISSUE #422: Integration with "Open Food Facts"
Wouldn't it be nice, if wger integrated with [Open Food Facts](https://world.openfoodfacts.org/)?

### PR DESCRIPTION
**Pull request by nikithamurikinati**
## Proposed Changes

  - partially resolves #422
  - created a script to import the open food facts ingredients into the ingredient database
  - added some additional fields such as brand, source, source_url, common_name, last_imported, category, and code (source id)
  - added setup for ingredient category, but do not import it from the open food facts script yet
  - import script should be run on production database (will take a long time)
  - still todo: update tests for updated ingredient model, write update script for open food facts, add images, finish category import

### Please check that the PR fulfills these requirements

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] New python code has been linted with with flake8 (``flake8 --config .github/linters/.flake8``)
and isort (``isort``) 
- [x] Added yourself to AUTHORS.rst

### Other questions

* Does this PR introduce a breaking change such as a database migration? (i.e.
what changes might users need to make in their running application due to
this PR?)
Yes, migrations should be rerun to add the new ingredient fields

### CODE REVIEW

File: extras/scripts/create_ingredients_from_foodfacts.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    common_name = product.get('generic_name', None)
+    brand = product.get('brands', None)
+
+    source_name = "Open Food Facts"
+    source_url = f'https://world.openfoodfacts.org/api/v0/product/{code}.json'
+
+    lang_object = Language.objects.create(
+        short_name=lang,
+        full_name=langs[lang]
+    )
```
Comment (rolandgeider) [inline extras/scripts/create_ingredients_from_foodfacts.py:50]: Can you add the changes I mentioned on discord?

**Comment by rolandgeider** [comment]
Looks good

File: extras/scripts/create_ingredients_from_foodfacts.py (hunk 1 above)
Comment (nikithamurikinati) [reply extras/scripts/create_ingredients_from_foodfacts.py:50]: Yep sorry, seems like an older version got uploaded for some reason!

Comment (harlenesamra) [reply extras/scripts/create_ingredients_from_foodfacts.py:50]: Changes got reverted somehow. Fixing now!

**Comment by rolandgeider** [comment]
Thanks for the PR! I want to do some more testing before I run this on the production database (on a test run I noticed how there are almost-but-not-quite duplicates of products) and also need to think about how to distribute the fixtures after this

**Comment by rolandgeider** [comment]
I have merged this manually, as I had been experitenting a bit with it, closing this PR
